protos/capture_data.proto

```diff
@@ -13,3 +13,33 @@ message FunctionStats {
   uint64 min_ns = 4;
   uint64 max_ns = 5;
 }
+
+message FunctionInfo {
```

Comment:
FunctionInfo was introduced only to save hooked functions (that are stored in GSelectedFunctionsMap). Fields `file`, `line`, `type` and `pretty_name` aren't used in context of hooked functions, but the `stats` field is only useful for hooked functions (in other cases it's not filled).